pkg/asset/stock/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package stock defines a stock type that holds the instanciated object of the assets.
```

Comment:
nit: My aspell dictionary [and wiktionary][1] have "instantiated" but not "instanciated".

[1]: https://en.wiktionary.org/wiki/instantiated#English